tests/FakeItEasy.Tests.Approval/ModuleInitializer.cs

```diff
@@ -0,0 +1,12 @@
+namespace FakeItEasy.Tests.Approval
+{
+    using System.Runtime.CompilerServices;
+    using VerifyTests;
+    using VerifyTests.DiffPlex;
+
+    public static class ModuleInitializer
+    {
+        [ModuleInitializer]
+        public static void Initialize() => VerifyDiffPlex.Initialize(OutputType.Compact);

```

Comment:
Fancy! I don't think I ever used a module initializer before.
But maybe this could be done in the test class static constructor? It would feel a bit less "magic"...